specification.md

```diff
@@ -1942,7 +1960,7 @@ If events defined in event-based conditions do not arrive before the states `eve
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
 | name | Data condition name | string | no |
-| condition | JsonPath expression evaluated against state data. True if results are not empty | string | yes |
+| condition | jq expression evaluated against state data. Must evaluate to true or false | string | yes |
```

Comment:
According to the jq docs:

> false and null are considered "false values", and anything else is a "true value".

Should this follow that standard?  If not, what should happen to a workflow where some non-boolean value is returned by the filter.

Reply:
Good point. I think it would make more sense then to say
"jq expression evaluated against state data. Condition is considered as true if the rest of the expression is true. If the result of the expression is anything else, the condition is considered false."